build.gradle

```diff
@@ -43,6 +43,7 @@ repositories {
 }
 
 dependencies {
+    implementation 'com.google.inject:guice:5.1.0'
```

Comment:
Since this is adding a dependency, what is the jar size cost?

Reply:
> jar (759 KB)
https://mvnrepository.com/artifact/com.google.inject/guice/5.1.0


I mean jar size shouldn't matter. If it does we should fix that.
~ Tazz

Reply:
Jar size is important, it is the size of the application after all.

Furthermore I am not talking about the dependency jar, I am considering the dependencies final impact to the launcher jar. 

For example, dbus lead to an approximately 5mb increase in file size due to its implied dependencies.